ldregistry/templates/main/_not-found.vm

```diff
@@ -1,6 +1,6 @@
   <div class="jumbotron">
     <h1>$msg['error.notFound.heading']</h1>
     <p>
-      $msg.get('error.notFound.note', $target, $lib.escapeHtml($target))
+      $msg.get('error.notFound.note')
```

Comment:
Perhaps https://github.com/UKGovLD/registry-config-base/blob/master/ldregistry/config/language/messages/en.properties#L80 should also be updated to remove the placeholder to something like `error.notFound.note=Sorry, the requested URI was not found on this site.`?

Reply:
Thanks Mike, yes, now done. I had that on a list to check on but juggling too many things at once.